tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "paths": {
       "@/shared/*": ["./packages/shared/src/*"],
       "@/swap/*": ["./packages/swap/src/*"],
-      "@/sdk/*": ["./packages/sdk/src/*"],
```

Comment:
we dont want other packages than the examples (eg the `swap` package) to import files from the sdk

Reply:
(this is a follow up for https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/1274)